test/black-box/escrow/EscrowAddCommand.test.ts

```diff
@@ -82,9 +259,20 @@ describe('EscrowAddCommand', () => {
         expect(result.type).toBe(testData.type);
         expect(result.Ratio.buyer).toBe(testData.ratio.buyer);
         expect(result.Ratio.seller).toBe(testData.ratio.seller);
+
+        // Double check the object was created with the new values.
```

Comment:
all service.create()'s will call service.findOne() in the end returning the created model, so double checking is propably not necessary.

Reply:
I understand that, but I think that within tests more checks is better than less.
Expect the impossible, etc